Objects/frameobject.c

```diff
@@ -66,6 +66,251 @@ get_arg(const _Py_CODEUNIT *codestr, Py_ssize_t i)
     return oparg;
 }
 
+typedef struct _codetracker {
+    unsigned char *code;
+    Py_ssize_t code_len;
+    unsigned char *lnotab;
+    Py_ssize_t lnotab_len;
+    int start_line;
+    int offset;
+    int line;
+    int addr;
+    int line_addr;
+} codetracker;
+
+static void reset(codetracker *tracker) {
```

Comment:
This isn't following [PEP 7](https://www.python.org/dev/peps/pep-0007/#code-lay-out) where the opening `{` for a function is on the line below the definition line as well as the scoping and return type (which you did to later).
```suggestion
static void
reset(codetracker *tracker)
{
```